File: backend/src/main/java/edu/cit/amihan/medibook/auth/dto/AuthResponse.java
package edu.cit.amihan.medibook.auth.dto;

import edu.cit.amihan.medibook.user.entity.Role;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AuthResponse {
    private String accessToken;
    private String refreshToken;
    private Long userId;
    private String username;
    private String fullName;
    private Role role;
}

File: mobile/app/src/main/java/edu/cit/amihan/medibook/feature/auth/model/AuthResponse.kt
package edu.cit.amihan.medibook.feature.auth.model

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    @SerializedName("accessToken") val accessToken: String? = null,
    @SerializedName("refreshToken") val refreshToken: String? = null,
    @SerializedName("userId") val userId: Long = 0L,
    @SerializedName("username") val username: String? = null,
    @SerializedName("fullName") val fullName: String? = null,
    @SerializedName("role") val role: String? = null
)
